apps/laboratory/tests/shared/fixtures/w3m-fixture.ts

```diff
@@ -30,6 +30,18 @@ export const testM = timingFixture.extend<ModalFixture>({
     await use(modalPage)
   }
 })
+export const testMEthers = timingFixture.extend<ModalFixture>({
```

Comment:
How is this different fromcthe default testM? / why does it need a special case?